src/lib/CPAuth/CPAuth.ts

```diff
@@ -0,0 +1,36 @@
+import { getAuthorityURLFromGSApiURL } from 'lib/CPAuth/util';
+import OAuth2, { IOAuth2Config } from 'lib/OAuth2/OAuth2';
+import { AppRoutes } from 'shared/constants/routes';
+
+const defaultConfig: IOAuth2Config = {
+  authority: getAuthorityURLFromGSApiURL(window.config.apiEndpoint),
+  clientId: 'zQiFLUnrTFQwrybYzeY53hWWfhOKWRAU',
+  clientSecret: 'TVHzVPin2WTiCma6bqp5hdxgKbWZKRbz',
```

Comment:
Right now these are the same as the ones from the `kubectl-gigantic` plugin. They will have bespoke values once we merge:
- https://github.com/giantswarm/installations/pull/1298
- https://github.com/giantswarm/dex-app/pull/13